modules/datagrid/72/launch/added/launch/infinispan_service_profiles.sh

```diff
@@ -68,7 +68,6 @@ function configure() {
          export DEFAULT_CACHE_OWNERS="2"
          export DEFAULT_CACHE_PARTITION_HANDLING_WHEN_SPLIT="DENY_READ_WRITES"
          export DEFAULT_CACHE_PARTITION_HANDLING_MERGE_POLICY="REMOVE_ALL"
-         export DEFAULT_CACHE_MEMORY_STORAGE_TYPE="off-heap"
```

Comment:
Is this correct? Due to limitations of the jvm in containers I thought storing entries on heap results in the cache having less capacity